Experience.html

```diff
@@ -21,7 +24,7 @@ <h2 class="aldi">permanent store assistant (ALDI) </h2>
              <li>prepare special buys  </li>
           </ul>
```

Comment:
Correct indentation looks like this

```
<nav>
  <h2>permanent store assistant (ALDI)</h2>
  <ul>
    <li>professional  customer service </li>
    ...
  </ul>
</nav>
```

Best to keep it consistent e.g. 2 spaces. Only need to indent if there is no closing tag on on previous line.